server/schedule/operator/create_operator.go

```diff
@@ -73,6 +74,19 @@ func CreateForceTransferLeaderOperator(desc string, cluster opt.Cluster, region
 		Build(kind)
 }
 
+// CreateEvictLeaderOperator creates an operator that transfers the leader from a source store to one of the given target stores
+// or the given target store when incompatible.
+func CreateEvictLeaderOperator(desc string, cluster opt.Cluster, region *core.RegionInfo, sourceStoreID uint64, targetStoreID uint64, targetStoreIDs []uint64, kind OpKind) (*Operator, error) {
```

Comment:
Can we use a more general name since it could be used in other scenes?

Reply:
Emm, in my first edition implementation, I directly modified `TransferLeader` to support multi targets. But then we found only evict leader needs it, and other ops only needs one target. So I created a seperate `EvictLeader`

Reply:
@rleungx Any updates about the naming? I've not come up with a good idea yet.

Reply:
No, but it needs to be general. ^ ^